articles/storage/storage-managed-disks-overview.md

```diff
@@ -144,7 +144,7 @@ For more information about Managed Disks, please refer to the following articles
 
 * [Attach a managed data disk to a Windows VM using PowerShell](../virtual-machines/windows/attach-disk-ps.md)
 
-* [Add a managed disk to a Linux VM](../virtual-machines/linux/quick-create-cli.md)
+* [Add a managed disk to a Linux VM](../virtual-machines/linux/add-disk#attach-a-managed-disk)
```

Comment:
@robinsh Thank you for your review.
I got your suggestion. By the way, I notice a little bit things, your provided link includes `linus`, but does it says about `linux`, doesn't it?